enterprise/api/v1/serializers.py

```diff
@@ -1406,3 +1406,16 @@ class EnterpriseCustomerUnlinkUsersSerializer(serializers.Serializer):
         required=False,
         default=False,
     )
+
+
+class EnterpriseCatalogQuerySerializer(serializers.ModelSerializer):
+    """
+    Serializer for the ``EnterpriseCatalogQuery`` model.
+    """
+
+    class Meta:
+        model = models.EnterpriseCatalogQuery
+        fields = '__all__'
+
+    # Parses from a dictionary to JSON
+    content_filter = serializers.JSONField(required=False)
```

Comment:
Distinction required due to default dictionary output without serializing in this format